analysis/metrics.py

```diff
@@ -311,13 +303,32 @@ class MetricUtility:
 
 
 @dataclass
-class PrivatePartitionSelectionMetrics:
-    """Stores aggregate metrics about partition selection."""
+class PartitionMetrics:
```

Comment:
I think `Partition(s)Info(rmation)` sounds better than `PartitionMetrics` since it can be distinguished from the other ones. And maybe we can think of something better later on.

Reply:
Agree, I've renamed.